aws-terraform/README.md

```diff
@@ -1,104 +1,113 @@
 Example:
 ```bash
-aws secretsmanager create-secret --name op-scim-dev/scimsession --secret-binary file:///path/to/scimsession --region <aws_region>
+aws secretsmanager create-secret --name op-scim/scimsession --secret-binary file:///path/to/scimsession --region <aws_region>
 aws secretsmanager describe-secret --secret-id op-scim/scimsession --region <aws_region>
-```  
-Custom KMS key can be specified by `--kms-key-id <kms_key_arn>`, in this case make sure deployed instances have access to that key, otherwise skip this option. If you don't specify key, then Secrets Manager defaults to using the AWS account's default CMK (the one named `aws/secretsmanager`).  
+```
+
+Custom Key Management Service key can be specified by `--kms-key-id <kms_key>`. Ensure deployed instances have access to that key. If you don't specify key, Secrets Manager will default to using the AWS account's default Customer Master Key (CMK, the one named `aws/secretsmanager`).
+
+3. Adjust `variables.tf`, `main.tf`, `providers.tf`, and `user_data/03-default-users.yml` as required, paying special attention to variables tagged with `CHANGE_ME`.
 
-3. Adjust `variables.tf`, `main.tf` and `providers.tf` as required.   
+4. Ensure that you've manually created a certificate for the subdomain you've specified in `variables.tf` in your AWS Certificate Manager. This is not automatically performed by the deployment script.
+
+5. Use the following `terraform` commands to deploy and verify your installation. From the `terraform/deploy/{testing,production}` directory:
 
-4. Use your favorite _terraform_ commands to deploy, verify, and troubleshoot it.
 ```bash
   terraform init
-  terraform plan -out=/tmp/op-scim.plan
-  # verify plan output
-  terraform apply /tmp/op-scim.plan
+  terraform plan -out=./op-scim.plan
+  terraform apply ./op-scim.plan
 ```
 
-5. Update/Create OP-SCIM endpoint configuration in Azure AD or Okta (IdP) using ALB URL as the endpoint and generated [_Bearer Token_](#Prepare-your-1Password-Account).
+6. Update/Create 1Password SCIM configuration in your Identity Provider using your generated [bearer token](https://github.com/1Password/scim-examples/tree/master/PREPARING.md) and newly-created subdomain.
+
+
+## Advanced
+
+### Logging
+
+All logs go to `syslog` (AWS EC2 instance OS). You can use the AWS System Manager to view logs for your instance.
 
-6. Application logs are available with the service start up. All logs go to `syslog` (AWS EC2 instance OS). You may consider configuring a "bastion" host or AWS System Manager or assign Public IP address to your instance, in order to be able to login to the deployed EC2 instance to view system logs.  
 
+### Upgrading and Redeployment
 
-## Deprovision (destroying) and Re-deploying
+You can destroy and redeploy the instance whenever you feel the need to. No permanent data is stored within the SCIM bridge. This is useful for upgrading your SCIM bridge with important bugfix or feature releases.
```

Comment:
The only permanent data is stored in the secret manager you reference below. It's a slight contradiction. I would reword slightly.